sahi/auto_model.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, Optional
+
+from sahi.utils.file import import_model_class
+
+MODEL_TYPE_TO_MODEL_CLASS_NAME = {
```

Comment:
I think this registry should be better as class variable inside AutoDetectionModel. Also, why don't save the classes directly instead of saving only their name?

Reply:
yes better move it inside the class.

saved only the names to avoid adding a new module import once a new detector model support is added. this way we only need to update a string (instead of adding module import).

can switch to saving directly the classes if its a better practice.

Reply:
Why don't you want to avoid imports? Are they too expensive (I doubt it) or some of them require installation of other packages? As for the better practice, this way you can't catch bugs e.g. incorrectly written class names would not be noticed until the interpreter tries to instantiate them during run time.